test/Tests.Infrastructure/ClusterTestBase.cs

```diff
@@ -390,6 +390,7 @@ protected async Task DisposeAndRemoveServer(RavenServer serverToDispose)
             List<IDictionary<string, string>> customSettingsList = null,
             bool watcherCluster = false)
         {
+            string[] allowedNodeTags = { "A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z" };
```

Comment:
Why do you need all this?

Reply:
I add the servers node tag to servers directory name
https://github.com/ravendb/ravendb/pull/10097/files#diff-214e21a3b1d9db763301cffa4d095f2dR567